pages/emailSender.tsx

```diff
@@ -165,95 +166,101 @@ const EmailSender: NextPage = () => {
 
   return (
     <Layout>
-    <ThemeProvider theme={theme}>
-      <StyledPageContainer>
-        <Typography variant="h3"> Email Sender </Typography>
-        <Divider />
-        <br />
-        <FormControl fullWidth>
-          <SectionContainer>
-            <StyledSubHeader variant="h5">1) Enter message</StyledSubHeader>
-            <br />
-            <StyledTextArea
-              aria-label="message-text-area"
-              placeholder="Paste in message"
-              onChange={(e) => setMessage(e.target.value)}
-              minRows={20}
-            />
-          </SectionContainer>
+      <ThemeProvider theme={theme}>
+        <StyledPageContainer>
+          <Typography variant="h3"> Email Sender
+          </Typography>
+          <Divider />
+          <br />
+          <Typography variant="body1">
+            <Link href="/help" underline="hover">
+              Help
```

Comment:
Maybe change this text to be a big more descriptive like 'Help page' or 'Email sender help page'